Yoast/wordpress-seo
# Change copy for new issue notification

## Summary

This PR can be summarized in the following changelog entry:

* Changes the notification message when a new issue has been found

## Relevant technical choices:

* Not applicable

## Test instructions

This PR can be tested by following these steps:

* Go to Reading -> discourage Search Engine Visibility
* Save changes and see the notification message in the admin bar appear

## Quality assurance


Fixes #4673
